sdk/identity/Azure.Identity/src/Azure.Identity.csproj

```diff
@@ -8,7 +8,6 @@
     <PackageTags>Microsoft Azure Identity;$(PackageCommonTags)</PackageTags>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <NoWarn>$(NoWarn);3021;AZC0011</NoWarn>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
Why is this needed?

Reply:
It's not! It was removed unnecessarily and has been undone.